apiserver/src/test/java/org/dependencytrack/persistence/jdbi/MetricsDaoTest.java

```diff
@@ -56,25 +55,51 @@ public void after() {
         super.after();
     }
 
+    @Test
+    public void testCreatePortfolioMetrics() {
+        var metrics = metricsDao.createPortfolioMetrics(1, 0, Instant.now(), Instant.now(),
+                0, 0, 0, 0, 0, 0, 0, 0, 0);
+        assertThat(metrics).isNotNull();
+        assertThat(metrics.getProjects()).isEqualTo(1);
+    }
+
+    @Test
+    public void testCreateProjectMetrics() {
+        final var project = qm.createProject("acme-app", null, "1.0.0", null, null, null, null, false);
+        var metrics = metricsDao.createProjectMetrics(project.getId(), 0, Instant.now(), Instant.now(),
+                0, 0, 0, 0, 0, 1, 0, 0);
+        assertThat(metrics).isNotNull();
+        assertThat(metrics.getSuppressed()).isEqualTo(1);
+    }
+
+    @Test
+    public void testCreateDependencyMetrics() {
+        final var project = qm.createProject("acme-app", null, "1.0.0", null, null, null, null, false);
+        var component = new Component();
+        component.setProject(project);
+        component.setName("Acme Component");
+        component.setVersion("1.0");
+        qm.createComponent(component, false);
+
+        var metrics = metricsDao.createDependencyMetrics(component.getId(), project.getId(), Instant.now(),
+                Instant.now(), 1, 0, 0, 0, 0, 0, 0);
+        assertThat(metrics).isNotNull();
+        assertThat(metrics.getCritical()).isEqualTo(1);
+    }
```

Comment:
We probably won't need tests for these since they're already only used in tests.